chapters/exploitation-techniques/shellcodes/drills/06-challenge-binsh-shellcode/src/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gcc:latest AS build
+WORKDIR /app
+
+# Runtime stage
+COPY vuln.c .
+RUN gcc -o vuln vuln.c
+
+WORKDIR /app
+COPY --from=build /app/vuln .
+
```

Comment:
Why is this required? You copy from `build` stage to, again, `build` stage. It doesn't have any effect.